tests/ctst/steps/azureArchive.ts

```diff
@@ -368,6 +401,20 @@ When('i restore object {string} for {int} days', async function (this: Zenko, ob
     await S3.restoreObject(this.getCommandParameters());
 });
 
+When('i run sorbetctl to retry failed restore for {string} location', function (this: Zenko, location: string) {
+    const backbeatUUID = process.env.UUID;
```

Comment:
Do we really need that as an environment variable? The world parameters can be extended with this uuid (and maybe with a more meaningful name), what do you think?